bucket/pass-winmenu-nogpg.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": "1.9",
+    "homepage": "https://github.com/Baggykiin/pass-winmenu",
+    "description": "A simple, easy-to-use password manager for Windows that follows the philosophy of (and is compatible with) the Linux password manager [pass](https://www.passwordstore.org/).",
+    "license": "MIT",
+    "url": "https://github.com/Baggykiin/pass-winmenu/releases/download/v1.9/pass-winmenu-nogpg.zip",
+    "hash": "466436be3dfa5e250b2739ec5e3358d351a7b7c5e2f4564ce335592b36abb670",
+    "bin": "pass-winmenu\\pass-winmenu.exe",
```

Comment:
```suggestion
    "extract_dir": "pass-winmenu",
    "bin": "pass-winmenu.exe",
```

Reply:
I assume bin becomes `pass-winmenu.exe` when I add `extract_dir`, right?

Reply:
Yes. As in suggestion:

1 line removed, 2 added,

![S](https://i.imgur.com/TwEFtVg.png)

Reply:
Doh; my bad